tests/conftest.py

```diff
@@ -150,6 +156,11 @@ def _bdb(_setup_database, _configure_bigchaindb):
     flush_db(conn, dbname)
 
 
+@pytest.fixture
+def _genesis(_bdb, genesis_block):
```

Comment:
Oh, the reasoning behind the name was that if you are going to have a genesis block, anything that sits under is implicit.

A similar reasoning can be found in the marker name `bdb`, as it could more accurately be called `bdb_with_config` since the config setup is a prerequisite to the database setup.

I thought that once one knows what the marker does, then it is all good, and from that point onward, the simpler the name the better, as one is then focusing on writing tests. If one needs information about what the marker does, one can use the `pytest --marker`, which reminds me that I forgot to add the marker in the config. :smile: